packages/braintree-utils/.eslintrc.json

```diff
@@ -5,7 +5,8 @@
         {
             "files": ["*.ts", "*.tsx"],
             "rules": {
-                "@typescript-eslint/naming-convention": "off"
+                "@typescript-eslint/naming-convention": "off",
+                "@typescript-eslint/no-explicit-any": "off"
```

Comment:
It might be better to remove this rule (we can keep it for spec files, but I on't think it's a good idea to set it for all ts files in package)

Reply:
We have the same rule in another packages for `ts` files.I went in the same way

Reply:
It is probably a mistake. Lets remove it if